data-service/src/main/java/com/amazonaws/blox/dataservice/mapper/EnvironmentMapper.java

```diff
@@ -29,9 +30,9 @@
   @Mapping(source = "cluster", target = "instanceGroup.cluster")
   Environment toEnvironment(EnvironmentDDBRecord environmentDDBRecord);
 
-  @Mapping(source = "instanceGroup.cluster", target = "cluster")
-  EnvironmentTargetVersionDDBRecord toEnvironmentTargetVersionDDBRecord(Environment environment);
+  EnvironmentTargetVersionDDBRecord toEnvironmentTargetVersionDDBRecord(
```

Comment:
This mapper is throwing a bunch of warnings because of umapped properties:

```
:data-service:compileJava
/home/travis/build/blox/blox/data-service/src/main/java/com/amazonaws/blox/dataservice/mapper/EnvironmentMapper.java:28: warning: Unmapped target properties: "recordVersion, attributes".
  EnvironmentDDBRecord toEnvironmentDDBRecord(Environment environment);
                       ^
/home/travis/build/blox/blox/data-service/src/main/java/com/amazonaws/blox/dataservice/mapper/EnvironmentMapper.java:31: warning: Unmapped target property: "deploymentConfiguration".
  Environment toEnvironment(EnvironmentDDBRecord environmentDDBRecord);
              ^
/home/travis/build/blox/blox/data-service/src/main/java/com/amazonaws/blox/dataservice/mapper/EnvironmentMapper.java:33: warning: Unmapped target property: "recordVersion".
  EnvironmentTargetVersionDDBRecord toEnvironmentTargetVersionDDBRecord(
                                    ^
3 warnings
```

Could we add the annotations to ignore those to get rid of it? If they will soon be mapped, that's also a good spot to add a TODO:

```java
@Mappings({
   @Mapping(source = "recordVersion", ignore = true),
   // TODO: Map attributes
   @Mapping(target = "attributes", ignore = true)
})
```
... etc.